src/application/can_handler/can_handler.c

```diff
@@ -0,0 +1,80 @@
+#include "can_handler.h"
+
+QueueHandle_t busQueue_rx;
+QueueHandle_t busQueue_tx;
+
+can_callback_t callback_map[30];
```

Comment:
```suggestion
static can_callback_t callback_map[30] = {0};
```
use `static` as per BARR-C:2018 1.8a, and initialize to zeros

Initializing this to zeros is important!!! The code below assumes that an unregistered callback pointer will have the value of `NULL`, but without initialization this is not defined behaviour!

Reply:
Wouldn't it be more clear to initialize to `NULL`?

Reply:
As in `= {NULL}`? Yeah, I agree &mdash; updated the suggestion